pages/common/indent.md

```diff
@@ -11,6 +11,6 @@
 
 `indent --gnu-style {{path/to/source.c}} -o {{path/to/indented_source.c}}`
 
-- Format C/C++ source according to the style of Kernigan & Ritchie (K&R), no tabs, 3 spaces per indent, and wrap lines at 120 characters:
+- Format C/C++ source according to the style of Kernighan & Ritchie (K&R), no tabs, 3 spaces per indent, and wrap lines at 120 characters:
```

Comment:
Nice one. :+1: :sunglasses:

Reply:
Yeah, I particularly enjoyed fixing this one as well ;)

Reply:
> no tabs, 3 spaces per indent

Why would anybody want to use that? 🤣

Reply:
Idk — Google redirects me to "Kernighan" too. 🤷🏻‍♂️

Reply:
I knew I'd heard that name somewhere before. Brian Kernighan co-authored the first book on the C programming language, along with Dennis Ritchie. I remember seeing his name on a book in my Academic Support Tutor's office I think, and also on a picture in a slide deck in a lecture. [According to Wikipedia](https://en.wikipedia.org/wiki/Brian_Kernighan), Brian Kernighan did not have a hand in designing C in the first place though.

Reply:
> > no tabs, 3 spaces per indent
> 
> Why would anybody want to use that? 🤣

I work on a legacy code base that uses 3-space indentation. I think it's a compromise between 2 and 4. 😜

Reply:
Believe it or not, he had some hand in the birth of Golang, although I forget what (see [here](https://github.com/golang/go/commits/master?after=a2c07a9a1a445c28fa4db9bd1472b28cad3ca818+48070&branch=master))